drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -229,7 +229,8 @@ def custom_case_notes(...)
     end
 
     def files(**args)
-      resolve_files(**args)
+      # Exclude files that have been uploaded by custom assessments
+      resolve_files(object.files.left_joins(:custom_data_element).merge(Hmis::Hud::CustomDataElement.where(id: nil)), **args)
```

Comment:
This is equivalent to `.where.missing(:custom_data_element)`! learned that one recently, I think it's neat

```rails
> Hmis::File.where.missing(:custom_data_element).count
  Hmis::File Count (3.0ms)  SELECT COUNT(*) FROM "files" LEFT OUTER JOIN "CustomDataElements" "custom_data_element" ON "custom_data_element"."DateDeleted" IS NULL AND "custom_data_element"."value_file_id" = "files"."id" WHERE "files"."type" = $1 AND "files"."deleted_at" IS NULL AND "custom_data_element"."id" IS NULL  [["type", "Hmis::File"]]

> Hmis::File.left_joins(:custom_data_element).merge(Hmis::Hud::CustomDataElement.where(id: nil)).count
  Hmis::File Count (3.3ms)  SELECT COUNT(*) FROM "files" LEFT OUTER JOIN "CustomDataElements" ON "CustomDataElements"."DateDeleted" IS NULL AND "CustomDataElements"."value_file_id" = "files"."id" WHERE "files"."type" = $1 AND "files"."deleted_at" IS NULL AND "CustomDataElements"."DateDeleted" IS NULL AND "CustomDataElements"."id" IS NULL   [["type", "Hmis::File"]]
```